### PR TITLE
Merge current master with docker branch

### PR DESCRIPTION
**Pull request by benjwadams**

### CODE REVIEW

File: contrib/init/50_config [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eu
+
+: ${CRON_STRING:=0 * * *}
+
+echo 'PATH=/opt/conda/bin:$PATH' > /etc/crontab
+echo "${CRON_STRING} harvester /opt/conda/bin/sensorml2iso -s http://sdf.ndbc.noaa.gov/sos/server.php --verbose --response_formats='text/csv,text/xml;schema=\"ioos/0.6.1\"'  --output_dir=/srv/iso/ndbc 2>&1 | /usr/bin/logger -t sensorml2iso" >> /etc/crontab
+echo "${CRON_STRING} harvester /opt/conda/bin/sensorml2iso -s http://opendap.co-ops.nos.noaa.gov/ioos-dif-sos/SOS --verbose --response_formats='text/csv,text/xml;subtype=\"om/1.0.0/profiles/ioos_sos/1.0\"' --output_dir=/srv/iso/co-ops 2>&1 | /usr/bin/logger -t sensorml2iso" >> /etc/crontab
```
Comment (mwengren) [inline contrib/init/50_config:9]: Not sure this is appropriate for the general module.  I'm not a Docker pro, but won't this cause the default module when run via Docker to run these sensoml2iso commands?

**Review by mwengren — CHANGES_REQUESTED** [review]
@shane-axiom Any feedback on this?

File: contrib/init/50_config (hunk 1 above)
Comment (daf) [reply contrib/init/50_config:9]: This crontab configuration would be more in place in a specifically derived docker image from what should be a generic base.  The old approach using `ENTRYPOINT` yields a very flexible usage scenario: using `sensorml2iso` without having to "install" it in the traditional sense.

**Comment by lukecampbell** [comment]
It's not this shouldn't go into master

**Pull request closed without merging**